examples/online-store/settings.gradle

```diff
@@ -1,12 +0,0 @@
-

```

Comment:
I removed this because I didn't see why it was needed, and mlGradleVersion isn't defined anywhere.